helm/alloy/values.yaml

```diff
@@ -1,7 +1,15 @@
+# Available values options are available at https://github.com/grafana/alloy/blob/main/operations/helm/charts/alloy/values.yaml
 global:
   image:
     registry: gsoci.azurecr.io
 
+networkPolicy:
```

Comment:
Should that not be under alloy?

Reply:
Also, i'm fine with it but could you also push those networkpolicies upstream and link the pr hère?

Reply:
I can push this upstream, so later when this is hopefully accepted and merged upstream those values should be moved under `alloy`